Makefile

```diff
@@ -10,14 +10,11 @@ INTEGRATION=internal/integration
 TIMEOUT=5m
 
 ADLR_SRC=pkg
-ADLR_MAIN=adlrtool
-MOCK=mockery
-MOCKS=internal/mocks
+ADLR_MAIN=adlr-cli
 
 BIN=bin
 SCRIPTS=sh
 
-LICENSELOCK=$(ADLR_MAIN)/license.lock
```

Comment:
I removed the license lock code. The final file produced by the adlr-cli tooling is now `verified-licenses.json` (or whatever custom name you choose).